src/rpm/headers/header.rs

```diff
@@ -237,6 +238,18 @@ where
             })
     }
 
+    pub(crate) fn get_entry_number_data(&self, tag: T) -> Result<i64, RPMError> {
+        let entry = self.find_entry_or_err(&tag)?;
+        entry
+            .data
+            .as_number()
+            .ok_or_else(|| RPMError::UnexpectedTagDataType {
+                expected_data_type: "numeric",
```

Comment:
No such type exists officially, but I'm not sure how else to express that it could be any numeric type

Reply:
I think it boils down to, if it's allowed to change the number representation or if they're fixed per tag as given by the specification (well, if there was one..). So I think if it works with the defacto impl a numeric is an abstraction over all int types so this is fine, it's just an error right?

Reply:
FWIW, librpm has this, the function is `rpmtdGetNumber` and it just returns a uint64 regardless of what the tag type was.

Reply:
Interesting. If that's used consistently within rpm, we could technically make the serialized version whatever type we'd want to.

Reply:
Let's call it `"scalar"` for the sake of consistency